### PR TITLE
add opensuse_leap.15.2

### PR DESCRIPTION
**Pull request by liewegas**
Just a copy of 15.1!

Also updated cobber.yml...

Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by liewegas** [comment]
@djgalloway is this right?

File: roles/testnode/vars/opensuse_leap_15.2.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  # used by workunits
+  - attr
+  - valgrind
+  - python-nose
+  - ant
+#  - iozone
+  ###
+  # used by the xfstests tasks
+  - libtool
+  - automake
```
Comment (tchaikov) [inline roles/testnode/vars/opensuse_leap_15.2.yml]: do we still need this?

File: roles/testnode/vars/opensuse_leap_15.2.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  - gettext-runtime
```
Comment (tchaikov) [inline roles/testnode/vars/opensuse_leap_15.2.yml]: we don't do i18n using gettext, right?

File: roles/testnode/vars/opensuse_leap_15.2.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  - lsb-release
+  - sysstat
+  - gdb
+  - make
+  - git
+  - python-configobj
+  # for running ceph
+  - libedit0
+#  - libboost_thread1_54_0
+  - libboost_thread1_66_0
```
Comment (tchaikov) [inline roles/testnode/vars/opensuse_leap_15.2.yml]: we don't use boost shipped by distro.

File: roles/testnode/vars/opensuse_leap_15.2.yml (hunk 1 above)
Comment (liewegas) [reply roles/testnode/vars/opensuse_leap_15.2.yml]: yes, for xfstests

**Comment by liewegas** [comment]
this file is 100% identical to the 15.1 file

i'm not sure about the cobbler part?

**Comment by djgalloway** [comment]
:disappointed: oops.. the vars file got deleted in https://github.com/ceph/ceph-cm-ansible/commit/659d0a5a17f0fe41b7178bd6b6bd29276fe517c7

@smithfarm had already taken care of this in https://github.com/ceph/ceph-cm-ansible/pull/536

There also needs to be an entry in https://github.com/ceph/ceph-cm-ansible/blob/master/roles/cobbler_profile/defaults/main.yml#L130

**Comment by smithfarm** [comment]
> this file is 100% identical to the 15.1 file

The 15.1 file has Python 2 packages in it. I don't think we should be using those. 

Can't we just reinstate the one from #536 that got accidentally deleted?

**Comment by djgalloway** [comment]
> Can't we just reinstate the one from #536 that got accidentally deleted?

+1 I'll take care of this

**Comment by liewegas** [comment]
thanks!

**Comment by djgalloway** [comment]
https://github.com/ceph/ceph-cm-ansible/pull/539

**Pull request closed without merging**